sql/analyzer/rules.go

```diff
@@ -66,4 +67,5 @@ var (
 	// ErrMisusedAlias is returned when a alias is defined and used in the same projection.
 	ErrMisusedAlias = errors.NewKind("column %q does not exist in scope, but there is an alias defined in" +
 		" this projection with that name. Aliases cannot be used in the same projection they're defined in")
+	ErrTupleProjected = errors.NewKind("unexpected tuple found, maybe remove the ()?")
```

Comment:
I would change the message to `unexpected tuple found in SELECT clause, tuples cannot be selected`. Because not always a tuple will be a tuple only as a result of having ().

Reply:
:+1: